UI/js/scriptForView.js

```diff
@@ -370,51 +549,258 @@ let offers = new Model([
+
+    let filters = document.querySelector(".filters");
+    filters.addEventListener('change', handleFilter);
+
+    function handleFilter(event) {
+        let filter = event.target.parentNode.parentNode;
+        filterConfig.dateFrom = filter.elements.dateFromFilter.value;
+        filterConfig.dateTo = filter.elements.dateToFilter.value;
+        filterConfig.vendor = filter.elements.vendorFilter.value;
+        if (filter.elements.hashtagsFilter.value.length !== 0) {
+            filterConfig.hashTags = filter.elements.hashtagsFilter.value.split(' ');
+        } else {
+            filterConfig.hashTags = [];
+        }
+        filterConfig.hashTags = filter.elements.hashtagsFilter.value.split(' ');
+        view.refreshPage(0, 10, filterConfig);
+    }
+
+    let loadMoreButton = document.querySelector(".load-more-button");
+    loadMoreButton.addEventListener('click', handleLoadMoreButton);
+
+    let shown = 10;
+
+    function handleLoadMoreButton() {
+        view.refreshPage(0, +10, filterConfig, shown);
+        shown += 10;
+    }
+
+    let logInOutButton = document.querySelector(".log-out");
+    logInOutButton.addEventListener('click', handleLogInOutButton);
+
+    function getLogInPage(){
+        return `
+            <p class="log-in-message">Authorization form</p>
+            <form class='log-in-form'>
+                <input type="text" class="login" name ="user" placeholder="Login">
+                <input type="text" class="password" name="password" placeholder="Password">
+                <button class="sign-in-button">Sign in</button>
+            </form>
+        `
+    }
+
+    function loadInfoLogInOut() {
+        if (view.getIsLogIn()) {
+            view.setIsLogIn(false);
+            view.setIsVendor(false);
+            view.setUsername('Guest');
+            view.refreshPage();
+        } else {
+            view.setPage(document.querySelector(".page"));
+            document.querySelector(".page").remove();
+            let newPage = document.createElement("page");
+            newPage.className = "log-in-page";
+            newPage.innerHTML = getLogInPage();
+            document.querySelector('.header').after(newPage);
+        }
+    }
+
+    function handleLogInOutButton() {
+        loadInfoLogInOut();
+        if (document.querySelector(".log-in-page") != null) {
+            document.forms[0].addEventListener('submit', handleSignInButton);
+        }
+    }
+
+    function handleSignInButton(event) {
+        event.preventDefault();
+        let form = event.target;
+        if (form.elements.user.value !== '' && form.elements.password.value !== '') {
+            view.setUsername(form.elements.user.value);
+            view.setIsLogIn(true);
+            view.setIsVendor(true);
+            form.removeEventListener('submit', handleSignInButton);
+            document.querySelector(".log-in-page").remove();
+            document.querySelector('.header').after(view.getViewPage());
```

Comment:
This also should be moved to view. Actually, all the logic that manipulates page content is supposed to be a incapsulated in view, here you need to just call required methods.